pkg/boot/fit/fit.go

```diff
@@ -49,14 +47,46 @@ func New(n string) (*Image, error) {
 	return &Image{name: n, Root: fdt}, nil
 }
 
+// ParseConfig reads r for a FIT image and returns a OSImage for each
+// configuration parsed.
+func ParseConfig(r io.ReadSeeker, verbose bool) ([]Image, error) {
```

Comment:
You might want a unit test for this.

Reply:
There was already a utest, but I added a utest for a bad image if that's what you meant.